src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceBase.cs

```diff
@@ -203,6 +203,44 @@ select nc.Title
                 return string.Empty;
             }
         }
+        protected string GetNocGroup2021(int? nocId, bool isFrench = false)
+        {
+            if (nocId == null || nocId == 0)
+            {
+                return string.Empty;
+            }
+
+            string codeStr = nocId.ToString().PadLeft(4, '0');
```

Comment:
Pad 5 or 4 ?

Reply:
This issue has been fixed in the latest commit.